examples/offline_inference/basic/basic.py

```diff
@@ -1,35 +1,77 @@
 # SPDX-License-Identifier: Apache-2.0
 # SPDX-FileCopyrightText: Copyright contributors to the vLLM project
 
+import argparse
+import os
+
+os.environ["PT_HPU_LAZY_MODE"] = "1"
+
 from vllm import LLM, SamplingParams
 
-# Sample prompts.
-prompts = [
-    "Hello, my name is",
-    "The president of the United States is",
-    "The capital of France is",
-    "The future of AI is",
-]
-# Create a sampling params object.
-sampling_params = SamplingParams(temperature=0.8, top_p=0.95)
+# Parse the command-line arguments.
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--model",
+    type=str,
+    default="Qwen/Qwen3-Next-80B-A3B-Instruct",
```

Comment:
suggest to keep the default model to the original "facebook/opt-125m", since this basic.py will be used by other test cases too.